src/ui/connection_profiles.ui

```diff
@@ -2297,14 +2276,28 @@ p, li { white-space: pre-wrap; }
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_3">
+              <property name="text">
+               <string>Port:</string>
+              </property>
+             </widget>
+            </item>
+            <item row="0" column="1" colspan="3">
+             <widget class="QLineEdit" name="profile_name_entry">
               <property name="font">
                <font>
                 <family>Bitstream Charter</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
+              <property name="readOnly">
```

Comment:
Oh, not something I touched in the commit I attached below - safest to remove these lines with the "plain text editor" option in Qt Creator! 😀